### ISSUE TITLE
SW Wanton Destruction not playing final cutscene

### ISSUE THREAD

**Issue by Q3BFG10K** [ZDoom/Raze#1079]
Win10-64, Raze 1.9.1, Classic Redux data from steam.

End cutscene is supposed to play after Zilla dies but it isn't playing. Save attached.

[save0000.zip](https://github.com/ZDoom/Raze/files/14847951/save0000.zip)

**Comment by madame-rachelle** [comment]
It seems to be playing for me, on the current master branch. Does it still malfunction with recent devbuilds?

https://devbuilds.drdteam.org/raze/

**Comment by JarDrh99** [comment]
Maybe is related to this problem:
https://github.com/ZDoom/Raze/issues/1040

Did you load the game after finishing the level (but before ending it)?
If yes, this seems to happen on all games.

**Comment by Q3BFG10K** [comment]
@JarDrh99, yes, sort of. I pressed a key which caused me to skip the final cutscene, so I quickloaded to watch it again, but it never played.

@madame-rachelle, you tested my save file and the cutscene played for you after loading it?
